frontend/src/data/config/MetricConfig.ts

```diff
@@ -459,6 +459,7 @@ export const METRIC_CONFIG: Record<DropdownVarId, VariableConfig[]> = {
           metricId: "death_ratio_age_adjusted",
           fullCardTitleName:
             "Age-adjusted risk of COVID-19 death compared to White (NH)",
+          chartTitle: "Risk of COVID-19 death compared to White (NH)",
```

Comment:
i think the word `in` is missing.... looks like @eriwarr  has been including it in the metric config properties, but maybe it'd be better to move it out of the config and into the title function. Either way, rn the chart title says "Risk of COVID-19 death compared to White (NH) the United States" (missing `in`)

Reply:
good call. I think it should be in the function too

Reply:
That's a good approach, I think we'll just have to update the metric config to remove the "in", and then change the chart title to   

```
  const chartTitle = `${
    metricConfig.chartTitle ? `${metricConfig.chartTitle} in`: ""
  } ${breakdown ? `${breakdown} in` : ""}`;
```